test/unit/alphabet/composite/composite_integration_test.cpp

```diff
@@ -25,14 +26,15 @@ using gapped_qualified_dna_phred42 = seqan3::gapped<qualified_dna_phred42>;
 using qualified_qualified_gapped_dna_phred42_phred42 = seqan3::qualified<qualified_gapped_dna_phred42, seqan3::phred42>;
 using gapped_alphabet_variant_dna_phred42 = seqan3::gapped<seqan3::alphabet_variant<seqan3::dna4, seqan3::phred42>>;
 
-// Some haessllihckeiten-tests
+// Some haessllihckeiten-tests <--???
```

Comment:
:see_no_evil: that was frustrated-me when I worked on this 2 years ago.
The review process clearly failed here :rofl:

Reply:
Didn't we already remove this in another PR? Are there multiple occurrences?

Reply:
:laughing: